jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/ApplyService.java

```diff
@@ -1390,4 +1398,55 @@ public void processCustomEntities(List<String> customResourceDefinitions, File r
             }
         }
     }
+
+    public static void applyNamespaceOrProjectIfPresent(ApplyService applyService, Collection<HasMetadata> entities) {
+        List<HasMetadata> projectOrNamespaceList = entities.stream()
+                .filter(e -> e instanceof Namespace || e instanceof Project)
+                .collect(Collectors.toList());
+
+        for (HasMetadata projectOrNamespace : projectOrNamespaceList) {
+            if (projectOrNamespace instanceof Namespace) {
+                Namespace ns = (Namespace) projectOrNamespace;
+                applyService.applyNamespace(ns);
+            } else if (projectOrNamespace instanceof Project) {
+                Project project = (Project) projectOrNamespace;
+                applyService.applyProject(project);
+            }
+        }
+    }
+
+    public static void applyEntities(ApplyService applyService, String fileName, Set<HasMetadata> entities, KitLogger serviceLogger,
```

Comment:
Take this comment and apply it here https://github.com/eclipse/jkube/pull/454/files#r524302202

If you pass an instance of the enclosing class, then it really makes no sense to make this method static